src/hooks/web3Context.tsx

```diff
@@ -135,7 +135,8 @@ export const Web3ContextProvider: React.FC<{ children: ReactElement }> = ({ chil
 
     const connectedProvider = new Web3Provider(rawProvider, "any");
     setProvider(connectedProvider);
-    const connectedAddress = await connectedProvider.getSigner().getAddress();
+    const connectedAddress = await connectedProvider.getSigner().getAddress().catch(console.error);
+    if (!connectedAddress) return;
```

Comment:
not the best solution, but stops the app from fully breaking when wallet is locked